presto-cache/src/main/java/com/facebook/presto/cache/basic/BasicCachingFileSystem.java

```diff
@@ -0,0 +1,530 @@
+/*
```

Comment:
NOTE this class is simply renamed from the original `CachingFileSystem.java`

Reply:
We should rename this? How about HiveCachingFileSytem?